src/examples/algorithms/HelloSP_TP.cpp

```diff
@@ -1,96 +0,0 @@
-/* ---------------------------------------------------------------------
```

Comment:
It did the same what this example does, but worse, imho. 
I created the file to be a `profiling benchmark`, a functionality which I've moved to the anomaly example now. What do you think?